.github/actions/lint-front-matter/linters/all-files.js

```diff
@@ -0,0 +1,15 @@
+const getYamlFrontMatter = require('../utils/get-yaml-front-matter.js.js');
+const checkHasProperty = require('../rules/has-property.js/index.js.js');
```

Comment:
This require path seems wrong. It has an extra suffix?

Reply:
Actually all the other files are `.js.js` too. Is there a reason?

Reply:
Hm. I think I may have told VS Code that it was ok for it to auto update my imports when i dragged a folder and perhaps it flubbed it.